models/attribute/src/attribute-draft/generator.ts

```diff
@@ -0,0 +1,18 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TAttributeDraft } from '../types';
+
+/**
+ * This model does not have a corresponding REST entity,
+ * as `AttributeDraft` doesn't exist.
+ *
+ * It's been created specifically for GraphQL transformations
+ * when needing `ProductAttributeInput`.
+ */
+const generator = Generator<TAttributeDraft>({
+  fields: {
+    name: fake((f) => f.lorem.slug(3)),
+    value: null,
```

Comment:
The documented type for this field in Impex is `String`, but in [this Slack thread](https://commercetools.slack.com/archives/C010MT0L58D/p1677163802757049), Yann mentions that it's in fact effectively the same as the HTTP `Attribute` representation (notably, it's basically `any` or `Json`)